src/CMakeLists.txt

```diff
@@ -26,6 +26,13 @@ if(Boost_UNIT_TEST_FRAMEWORK_FOUND)
     list(APPEND CUKE_SOURCES drivers/BoostDriver.cpp)
 endif()
 
+if(Qt5TEST_FOUND)
+    qt5_wrap_cpp(MOC_FILE ../include/cucumber-cpp/internal/drivers/QtTestDriver.hpp)
+    list(APPEND CUKE_SOURCES ${MOC_FILE})
+    list(APPEND CUKE_SOURCES drivers/QtTestDriver.cpp)
+    list(APPEND CUKE_DEP_LIBRARIES ${QT_LIBRARIES})
```

Comment:
Please only add `Qt5::Test`, the rest isn't necessary.

Reply:
right